modules/cnf-modifying-and-applying-the-default-profile.adoc

```diff
@@ -0,0 +1,15 @@
+// CNF-950 4.7 Modifying and applying the default profile
+// Module included in the following assemblies:
+//
+// *cnf-provisioning-and-deploying-a-distributed-unit.adoc
```

Comment:
* scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc

Reply:
Approved.